src/components/Home/index.tsx

```
@@ -57,6 +57,18 @@ const Index = (): JSX.Element => {
                 GameCI
               </a>
             </Card.Item>
+            <Card.Item>
+              🎮{' '}
+              <a href="https://minis.gg" target="_blank">
+                Minis.gg
```

Biome (2.3.14)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)